### ISSUE TITLE
Feature Request: "Fast Search" + Option to "Focus Search Bar on New Tab"

### ISSUE THREAD

**Issue by ashrafhadden** [brave/browser-android-tabs#496]
On Android Brave I find myself constantly opening new tabs in the browser to search something new and it would be great to eliminate the number of taps to get there (I’m an efficiency freak).

2 solutions…

“Fast Search” Feature

In writing this post and meditating on the issue of search speed I realized that long-pressing on the tab button itself (the small box on the upper-right of the UI showing the number of tabs) would be the fastest and most intuitive way to open a new tab with the search bar already focused and ready to type in your search terms. Right now long-pressing simply reveals a small label reading “Open tabs” which is obviously just an arbitrary placeholder. This would be a really slick feature that could be advertised on both Brave’s website and the Google Play store (in both the description features section and in the featured images).

“Focus Search Bar on New Tab” Option

Having an option that automatically focuses the Search Bar upon opening a new tab would be a nice little feature. I envision a checkbox under Settings>Accessibility that reads “Focus Search Bar on New Tab”. Back when I was on iPhone there were Jailbreak tweaks that achieved the same thing. It’s a really small thing but having little features like this would help set Brave apart from Chrome and other competing browsers.

**Comment by edmundlaugasson** [comment]
I totally agree and found current issue in searching solution for that. In Vivaldi Browser there is F2 shortcut key what brings a floating search window to search among opened tabs and even history. That is very comfortable. Searching among opened tabs and history is one of the big missing feature in Brave.
